pkg-manager/link-bins/src/index.ts

```diff
@@ -286,10 +287,25 @@ async function linkBin (cmd: CommandInfo, binsDir: string, opts?: LinkBinOptions
   // ensure that bin are executable and not containing
   // windows line-endings(CRLF) on the hashbang line
   if (EXECUTABLE_SHEBANG_SUPPORTED) {
-    await fixBin(cmd.path, 0o755)
+    if (await canFixBin(cmd.path)) await fixBin(cmd.path, 0o755)
+    else globalWarn(`Skipped fixing bin permissions of \`${cmd.path}\` because the file is not owned by the current user`)
   }
 }
 
+async function canFixBin (filePath: string): Promise<boolean> {
+  if (IS_WINDOWS) return true
```

Comment:
Skips windows, as I don't know if this problem happens on Windows machines too or not?